go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/packethost/packet-cli
 
 go 1.13
 
+replace github.com/packethost/packngo => ../packngo
```

Comment:
This will be removed when packngo has a release with metros (waiting for VLAN support)